internal/application/application.go

```diff
@@ -236,6 +236,11 @@ func Delete(ctx context.Context, cluster *kubernetes.Cluster, gitea GiteaInterfa
 		return err
 	}
 
+	err = cluster.Kubectl.CoreV1().Secrets(appRef.Org).Delete(ctx, fmt.Sprintf("%s-tls", appRef.Name), metav1.DeleteOptions{})
```

Comment:
Maybe we should do this instead? https://cert-manager.io/docs/usage/certificate/#cleaning-up-secrets-when-certificates-are-deleted

Reply:
I agree, but this will be done for all the cert generated by cert manager. Should be ok for us.

I'll see if it works , otherwise it'll have to go in a separate PR.

Reply:
I think it worked.